modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/DateTimeObjectFieldBusinessType.java

```diff
@@ -67,6 +67,11 @@ public String getDDMFormFieldTypeName() {
 		return DDMFormFieldTypeConstants.DATE_TIME;
 	}
 
+	@Override
+	public String getDDMFormFieldTypeName(ObjectField objectField) {
```

Comment:
If you really need to override this method, just do like other types implementing `getDDMFormFieldTypeName(boolean localized)`